circle.yml

```
@@ -8,7 +8,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/node:10
+      - image: circleci/node:18
```

```suggestion
      - image: cimg/node:lts
```

see https://circleci.com/developer/images/image/cimg/node